notebooks/official/matching_engine/sdk_matching_engine_create_stack_overflow_embeddings_palm.ipynb

```diff
@@ -0,0 +1,2028 @@
+{
```

Comment:
<p>I have not needed quotes when using the [tag] syntax</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/1876/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>

Reply:
<p>Some IDE's do need it unfortunately.</p>

Reply:
<p>I forget which one. Maybe VSCode, but I don't exactly remember.</p>